app/views/api/v2/common/show_hidden.json.rabl

```diff
@@ -0,0 +1,7 @@
+object @parameter
+
+if params["show_hidden"] == "true" && @object.present? && @object.editable_by_user?
+  attribute locals[:value].to_sym
```

Comment:
why not pass the field name as a symbol instead of passing as a string and calling `to_sym`?